JavaScript-Runtime-(JSRT)-Overview.md

```diff
@@ -85,9 +85,117 @@ All other supported ES2015 features do not require any setup to run in JSRT.
 
 Features based on promises, such as [ES2016/ES7 Async Functions](https://blogs.windows.com/msedgedev/2015/09/30/asynchronous-code-gets-easier-with-es2016-async-function-support-in-chakra-and-microsoft-edge/), would require setting up promises before usage. 
 
+## Modules
+ES6 Modules are supported in ChakraCore. However, the host needs to specify how to resolve a module specifier since this part is host-specific not runtime-related. JSRT provides experimental APIs to set up ES6 modules. This [PR](https://github.com/Microsoft/ChakraCore/commit/50767f9f33812b4a036794a6a98404145563409f) gives you an overview of the related APIs and an example implementation (see WScriptJsrt.cpp) in the ch test host. Better documentation will be provided in the future.
+
```

Comment:
For the PR you might want to link to the actual PR instead of the commit

https://github.com/Microsoft/ChakraCore/pull/1254